### PR TITLE
docs: Update installation steps

### PR DESCRIPTION
**Pull request by danielpacak**
Signed-off-by: Daniel Pacak <pacak.daniel@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,10 +74,15 @@ navigation menu:
 [lens]: https://github.com/lensapp/lens
 [starboard]: https://github.com/aquasecurity/starboard
 [starboard-crds]: https://github.com/aquasecurity/starboard#custom-security-resources-definitions
+[starboard-getting-started]: https://github.com/aquasecurity/starboard#getting-started
 
 [lens-installation]: https://github.com/lensapp/lens#installation
 [lens-releases]: https://github.com/lensapp/lens/releases
+[release-img]: https://img.shields.io/github/release/aquasecurity/starboard-lens-extension.svg?logo=github
```
Comment (simar7) [inline README.md:81]: I didn't know you could define anchors/variables like this in markdown. Does this work for any markdown flavour or just Github?

File: README.md [hunk 2]
```diff
@@ -3,34 +3,65 @@
...
-```
+1. Clone the source code to the `~/.k8slens/extensions/starboard-lens-extensions` directory and build the extension's package:
+   ```
+   $ mkdir -p ~/.k8slens/extensions
+   $ git clone https://github.com/aquasecurity/starboard-lens-extension.git ~/.k8slens/extensions/starboard-lens-extension
+   $ cd ~/.k8slens/extensions/starboard-lens-extensions
+   $ npm install && npm run build
+   ```
+2. Open Lens application and select **Lens** menu, and then click **Extensions** item, or press
+   <kbd>Shift</kbd> + <kbd>Command</kbd> + <kbd>E</kbd> to open the **Manage Lens Extensions** page.
```
Comment (simar7) [inline README.md:56]: TIL about the <kbd> tag!

File: README.md [hunk 3]
```diff
@@ -3,34 +3,65 @@
...
+3. If everything is fine, you'll see the `@aquasecurity/starboard-lens-extension` extension listed under
```
Comment (simar7) [inline README.md:57]:
```suggestion
3. If everything goes fine, you'll see the `@aquasecurity/starboard-lens-extension` extension listed under
```

File: README.md [hunk 4]
```diff
@@ -3,34 +3,65 @@
...
+4. If everything is fine, you'll see the `@aquasecurity/starboard-lens-extension` extension listed under
+   **Installed Extensions**. Click **Enable** to enable it.
+   ![](docs/images/extensions-ui-enable.png)
+5. Close the **Manage Lens Extensions** page and verify that the **Starboard** menu item shows up in the main menu.
+   ![](docs/images/starboard-extension-verify.png)
 
 ### From Source (Linux, macOS)
 
-Building from source is a bit of work, but currently it's the only way to install the latest version of the extension.
+Building from source is slightly more work, but is the best way to go if you want to test the latest (pre-release)
```
Comment (simar7) [inline README.md:43]:
```suggestion
Building from source is slightly more involved, but it's the best way to go if you want to test the latest (pre-release)
```

File: README.md [hunk 5]
```diff
@@ -3,34 +3,65 @@
...
+  initialized and how it generates security reports, which are then stored as custom resources.
+
+### From the Binary Releases
+
+Every [release][release] of Lens extension for Starboard provides the tarball-file that can be manually
+downloaded and installed. Alternatively, you can copy the download URL of the release artifact to paste it in the
+**Manage Lens Extensions** page.
+
+1. Download your desired version from GitHub [release][release] page or GitHub
+   [package registry](https://github.com/aquasecurity/starboard-lens-extension/packages/520430). Alternatively,
```
Comment (simar7) [inline README.md:29]: Could we make this link into a variable too?

**Review by simar7 — APPROVED** [review]
Left some minor nits but otherwise lgtm! 🦹🏻‍♂️

File: README.md (hunk 1 above)
Comment (danielpacak) [reply README.md:81]: According to https://www.markdownguide.org/basic-syntax/#reference-style-links it's not just GitHub flavour

File: README.md (hunk 5 above)
Comment (danielpacak) [reply README.md]: Sure. I've changed to reference style link

**Pull request merged**